bcipy/acquisition/tests/test_client.py

```diff
@@ -181,6 +192,43 @@ def test_offset(self):
 
         daq.cleanup()
 
+    def test_offset_column(self):
+        """Test offset calculation when a custom offset column is specified."""
+
+        col_name = 'PHOTODIODE'
+        channels = ['ch1', 'ch2', col_name]
+        sample_hz = 100
+        trigger_at = 10
+        num_records = 500
+        n_channels = len(channels) - 1
+
+        data = [
+            mock_record(n_channels) + [0 if (i + 1) < trigger_at else 1]
```

Comment:
It could be useful to define a function to produce this or set it as a class value

Reply:
Yeah, I still want to take a pass through the code and revisit mocked EEG data.